### DISCUSSION TITLE
putty not connected

### DISCUSSION THREAD

**Discussion by NARENDRA935** [learndevopsonline/learndevopsonline#1689] [Q&A] [answered]
<img width="908" alt="Screenshot 2023-12-21 115719" src="https://github.com/learndevopsonline/learndevopsonline/assets/153573686/a2a9e6b7-e01d-438b-90ca-f6d08dfb3592">
Hii  @r-devops ,
my putty not connected its showing no support authentication methods,please help me

**Comment by sreedharm07** [comment]
attach screenshot for better understanding of your error

**Reply by NARENDRA935** [reply]
yes,attached screenshot  please help me

**Reply by sreedharm07** [reply]
where is screenshot ?

**Comment by NARENDRA935** [comment]
screenshot in issue box

**Comment by NARENDRA935** [comment]
recently update the issue with screenshot

**Reply by sreedharm07** [reply]
dont open too many threads you are opening too many threads

and you forgot to give without key pair when you created instance thats y its expecting key
delete that instance and create new one with proper steps

**Reply by NARENDRA935** [reply]
plese share the steps today i am creating new instance first name ,next search devops-practice, next  t3.micro, key pair select without, security group allow-all then launch instance is there any wrong

**Reply by sreedharm07** [reply]
thats correct but you might miss key pair step 
so delete old one and create new with same steps which you mentioned now

**Comment by NARENDRA935** [comment]
yes i delete a old one create new instance then same issue there
<img width="908" alt="Screenshot 2023-12-21 115719" src="https://github.com/learndevopsonline/learndevopsonline/assets/153573686/c54cc77a-ae13-4247-a433-c6ddfef45cf7">

**Reply by sreedharm07** [reply]
ping your public ip

**Comment by NARENDRA935** [comment]
plese share the zoom link if u r available i will the screen

**Reply by sreedharm07** [reply]
dont open multiple threads

**Reply by sreedharm07** [reply, marked as answer]
https://us06web.zoom.us/j/88648359147?pwd=5acbvxGj6oNc5wJnBCupSQAZ0HrWRI.1 

join
